_targets.R

```diff
@@ -0,0 +1,83 @@
+library(targets)
+library(tarchetypes)
+library(dataRetrieval)
+library(EflowStats)
+library(tidyverse)
+library(lubridate)
+library(fasstr)
```

Comment:
Are all of these libraries needed in the main script? If they're only needed in some targets, you can specify them in a vector in `tar_option_set(packages = c(<package name>))`

Reply:
yes, most of those could probably go in the tar_option line.